.github/workflows/docs-deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Docs - Test and Deploy
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  build-test-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: Install NPM packages
+      run: npm ci
+
+    - name: Build style spec
+      run: |
+        npm run generate-style-spec
+        npm run generate-typings
+        npm run compile
+
+    - name: Build docs
+      run: |
+        cd ./docs
```

Comment:
Why do we need to `cd` into a folder and start `npm ci` and all that from that folder and not the main folder?

Reply:
Because I think it's unwise to pollute the style-spec package with the many many deps etc form the docs page.